### DISCUSSION TITLE
Automatic connection of gamepads - set Off

### DISCUSSION THREAD

**Discussion by MaxJumper** [darthcloud/BlueRetro#242] [Ideas]
Greeting.
I think it's worth adding a switch "1 is enough" or "2 is enough".
For example, a situation where there are several consoles in one room, and several BlueRetro, and several gamepads. To be able to limit the connection of all gamepads to one BlueRetro. Or some other mechanism for limiting the automatic connection of gamepads.

**Comment by darthcloud** [comment]
After first connection, pairing mode "SYNC" is turn off. If a controller connect pass that point it's because it's pair with BlueRetro and not another console.
